src/collectd.conf.pod

```diff
@@ -9386,11 +9380,38 @@ B<Note>: I<perf> metrics can't be collected if I<intel_rdt> plugin is enabled.
 =back
 
 =item B<PersistentNotification> B<true>|B<false>
+
 Override default configuration to only send notifications when there is a change
 in the lifecycle state of a domain. When set to true notifications will be sent
 for every read cycle. Default is false. Does not affect the stats being
 dispatched.
 
+=item B<Instances> B<integer>
+
+How many read instances you want to use for this plugin. The default is one,
+and the sensible setting is a multiple of the B<ReadThreads> value.
+
+This option is only useful then domains are specially tagged.
```

Comment:
Well, I'd rephrase. it is *most* useful when domains are tagged, but it could very much be useful if they aren't.
The domain tagging feature is supposed to be built on top of this feature, which predated it.